tensorflow_encrypted/protocol/securenn.py

```diff
@@ -73,11 +76,13 @@ def bitwise_xor(self, x: PondTensor, y: PondTensor) -> PondTensor:
     @memoize
     def msb(self, x: PondTensor) -> PondTensor:
         # NOTE when the modulus is odd then msb reduces to lsb via x -> 2*x
-        if x.backing_dtype.modulus % 2 != 1:
-            # NOTE: this is currently only for use with an odd-modulus CRTTensor
-            #       NativeTensor will use an even modulus and will require share_convert
-            raise Exception('SecureNN protocol assumes a ring of odd cardinality, ' +
-                            'but it was initialized with an even one.')
+
```

Comment:
should we remove this now or wait until share convert is fixed?

Reply:
I'll leave it commented out for now as we may still want to bench with exact relu and int64 despite getting wrong results